watsonx/runtime/src/main/java/io/quarkiverse/langchain4j/watsonx/bean/IAMError.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.langchain4j.watsonx.bean;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record IAMError(
+
+        @JsonProperty("errorCode") Code errorCode,
+        @JsonProperty("errorMessage") String errorMessage) {
```

Comment:
These should not be necessary if we configure the `ObjectMapper` to use camel case.

Reply:
The `IAMRestApi` uses the snake case for the "OK" response, but it is not true if there is an error.

Reply:
Understood, thanks